src/DelegateCall.sol

```diff
@@ -6,14 +6,17 @@ pragma solidity ^0.8.0;
 /// @custom:contact security@morpho.xyz
 /// @dev Low-level YUL delegate call library.
 library DelegateCall {
-    /// ERRORS ///
+    /* ERRORS */
```

Comment:
this won't be a problem with forge doc? btw, don't we want to setup an automated doc?

Reply:
Not it's the opposite. It's well handled